packages/devtools_app/lib/src/screens/network/network_request_inspector.dart

```diff
@@ -45,9 +45,14 @@ class NetworkRequestInspector extends StatelessWidget {
                     style: Theme.of(context).regularTextStyle,
                   ),
                 )
-              : AnalyticsTabbedView(
-                  tabs: _generateTabs(data),
-                  gaScreen: gac.network,
+              : ListenableBuilder(
+                  listenable: data,
+                  builder: (context, _) {
+                    return AnalyticsTabbedView(
```

Comment:
this worries me a bit. Read the dart doc for `AnalyticsTabbedView`. If we are rebuilding this unnecessarily, we will get a bunch of analytics events.